docs/install/linux.md

```diff
@@ -50,8 +48,8 @@ curl https://raw.githubusercontent.com/prysmaticlabs/prysm/master/prysm.sh --out
 ```sh
 ./prysm.sh beacon-chain
 ```
-
-It is also recommended to include the `--p2p-host-ip` and `--min-sync-peers 7` flags to improve peering. For advanced users that desire standard debugging tools found in the Busybox base image, append a `--debug` flag to enable them.
+> Advanced users: Please refer to the article on how to [improve P2P connectivity](../p2p-host-ip).  
```

Comment:
Maybe say like ' **Not getting enough peers?** refer to the [improving P2P connectivity](/docs/prysm-usage/p2p-host-ip) section of this documentation for helpful configuration tips.

Reply:
updated